src/scripts.js

```diff
@@ -81,38 +95,55 @@ function displayRecipeNames(recipeData) {
 }
 
 function displayAllNames() {
-  displayRecipeNames(recipeRepository.recipeData)
+  displayRecipeNames(recipeRepository.recipeData);
 }
 
 function showHomePage() {
   hide(recipePage);
   show(homePage);
+  show(searchContainer);
 }
 
 function viewRecipe(event) {
   hide(homePage);
+  hide(searchContainer);
   show(recipePage);
-  let selectedRecipeName = event.target.innerText
-  let selectedRecipe = allRecipes.filter(recipe => selectedRecipeName === recipe.name)
-  recipeNameBox.innerText = selectedRecipe[0].name
-  recipeDetailsBox.innerText = selectedRecipe[0].returnRecipeInstructions()
-  recipePriceList.innerText = selectedRecipe[0].getCostofRecipe()
+  let selectedRecipeName = event.target.innerText;
+  let selectedRecipe = allRecipes.filter(recipe => selectedRecipeName === recipe.name);
+  recipeNameBox.innerText = selectedRecipe[0].name;
+console.log(selectedRecipe[0])
+  recipeDetailsBox.innerHTML = '';
+  selectedRecipe[0].returnRecipeInstructions().forEach(instruction => {
+    recipeDetailsBox.innerHTML += `<p class='recipe-instructions'> ${instruction} </p></br>`
+  });
+// put code above into separate helper function
+  ingredientBox.innerHTML = '';
+  selectedRecipe[0].getIngredientNames().forEach(ingredient => {
+    ingredientBox.innerHTML += `<p class='recipe-ingredients'> ${ingredient} </p></br>`
+  });
+// put code above into separate helper function
+  priceListBox.innerHTML = '';
+  selectedRecipe[0].getCostOfIngredientsInDollars().forEach(cost => {
+  priceListBox.innerHTML += `<p class='ingredient-prices'> ${cost} </p></br>`
```

Comment:
Formatting for innerHTML- reviewed- great job! Looks fantastic on user story 2 selected recipe.